qa_tests/hazard/event_based/sc_utils.py

```diff
@@ -46,7 +46,7 @@ def joint_prob_of_occurrence(gmvs_site_1, gmvs_site_2, gmv, time_span,
         if gmv_close(gmv_site_1) and gmv_close(gmv_site_2):
             count += 1
 
-    prob = 1 - math.exp(- (float(count) / (time_span * num_ses)) * time_span)
```

Comment:
It is not related. It is not even a change:

``count / (time_span * num_ses) * time_span == count / num_ses``